src/public/index.ejs

```diff
@@ -15,6 +15,8 @@ <h2>
   </h2>
   <script src="./javascript/axios.min.js"></script>
   <script>
+    console.log('<%= GOOGLE_MAPS_API_KEY %>');
+
```

Comment:
So basically, just output `<%= GOOGLE_MAPS_API_KEY %>` anywhere you want to print that value. Don’t forget to throw quotes around it in JS because it’s a string!

Reply:
This also gives us the advantage of: if our key ever changed, then we’d just update our `.env` file locally but this repo would be none the wiser. And anyone trying to run this locally would have to supply their own key.